assets/base/i18n/en.json

```diff
@@ -177,7 +168,18 @@
   "mobile.join_channel.error": "We couldn't join the channel {displayName}. Please check your connection and try again.",
   "mobile.link.error.text": "Unable to open the link.",
   "mobile.link.error.title": "Error",
-  "mobile.login_options.choose_title": "Choose your login method",
+  "mobile.login_options.cant_heading": "Can't Log In",
+  "mobile.login_options.enter_credentials": "Enter your login details below.",
+  "mobile.login_options.gitlab": "GitLab",
+  "mobile.login_options.google": "Google",
+  "mobile.login_options.heading": "Log In to Your Account",
+  "mobile.login_options.none": "You can't log in to your account yet. At lease one login option must be configured. Contact your system Admin for assistance.",
+  "mobile.login_options.office365": "Office 365",
+  "mobile.login_options.openid": "Open ID",
+  "mobile.login_options.saml": "SAML",
+  "mobile.login_options.select_option": "Select a login option below.",
+  "mobile.login_options.separator_text": "or log in with",
+  "mobile.login_options.sso_continue": "Continue with",
   "mobile.managed.blocked_by": "Blocked by {vendor}",
   "mobile.managed.exit": "Exit",
   "mobile.managed.jailbreak": "Jailbroken devices are not trusted by {vendor}, please exit the app.",
```

Comment:
```suggestion
  "mobile.managed.jailbreak": "Jailbroken devices are not trusted by {vendor}. Login can't continue. Please exit the app.",
```

Reply:
@elias, Carrie submitted some grammar changes that are not related to this PR, but I did make them. See her other comments.

Regarding this suggestion, I'm not sure the we need the additional suggestion.  The i18n code is loaded in `apps/init/managed_app.ts`.  My assumption is tha that the app user wouldn't feel like they cannot login, but that the app can't load or initialize.  I'd suggestion not addint this suggestion. Your thoughts?

Reply:
yes, this is **NOT** login related, thank you!

Reply:
Also, @cwarnermm can we keep grammar and stuff reviews related to the PR, you can always submit a larger PR only for the grammar stuff in a separate one.

Also one thing that can massively help is to review the actual designs as we basically copy/paste the text that appears there.

Reply:
Absolutely! That additional wasn't intentional; I had mistakenly believed it to be part of the login updates. @jfrerich please feel free to dismiss any suggestions that aren't directly related to this login PR effort.

I welcome the opportunity to get involved earlier in the process by reviewing the designs prior to implementation. Please feel free to loop me in on the design phase of future efforts!

Reply:
If you are interested in the designs, they are located [here](https://www.figma.com/file/a9w2q3JnVvBwo2IScP8j3K/%E2%9C%85-Phase-1-Design?node-id=1544%3A91092).  I will make sure to ping you early in the code implementation process for grammar help :)  You might also reach out in the Mattermost `Feature: Gekidou & Mobile v2` channel to mention your desire to review the text of our new design!